File: Program17.cpp
#include<iostream>
using namespace std;

class Number
{
    private:
       int iNo;
    
    public:
       Number(int iValue)
       {
           this->iNo = iValue;
       }

       void Display()
       {
           int iCnt = 0;
           
           if(iNo < 0)  // Filter
           {
               cout<<"Please enter positive number";
               return;
           }

           for(iCnt = 1; iCnt<=iNo; iCnt++)
           {
               cout<<"Hello"<<endl;
           }
       }

};

int main()
{
    int iValue = 0;

    cout<<"Enter number"<<endl;
    cin>>iValue;

    Number nobj(iValue);

    nobj.Display();

    return 0;
    
}

File: Program17.java
import java.lang.*;
import java.util.*;

class Number 
{
    private int iNo;

    public Number(int iValue)
    {
        iNo = iValue;
    }

    public void Display()
    {
        int iCnt = 0;
        
        if(iNo < 0)  // Filter
        {
            System.out.println("Please enter positive number");
            return;
        }

        for(iCnt = 1; iCnt<=iNo; iCnt++)
        {
            System.out.println("Hello");
        }
    }
}
class Program17
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter number");
        int iValue = sobj.nextInt();
        
        Number nobj = new Number(iValue);

        nobj.Display();

    }
}
